packages/react-aria-components/stories/index.stories.tsx

```diff
@@ -1246,3 +1246,76 @@ export const LinkExample = () => {
     </Link>
   );
 };
+
+export const ToolbarExample = (props) => {
+  return (
+    <Toolbar {...props}>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><strong>B</strong></ToggleButton>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><div style={{textDecoration: 'underline'}}>U</div></ToggleButton>
+      <ToggleButton className={classNames(styles, 'toggleButtonExample')}><i>I</i></ToggleButton>
+      {/* radio does not work in toolbar yet, selection doesn't follow focus
+      <RadioGroup orientation="horizontal" className={styles.radiogroup} aria-label="Text Alignment">
+        <Radio className={styles.radio} value="left">Left</Radio>
+        <Radio className={styles.radio} value="center">Center</Radio>
+        <Radio className={styles.radio} value="right">Right</Radio>
+      </RadioGroup>
+      */}
+      {/* disabled buttons do not work in toolbar yet, the buttons cannot be focused
```

Comment:
cannot be? isn't that correct?

Reply:
based off the [aria example](https://www.w3.org/WAI/ARIA/apg/patterns/toolbar/examples/toolbar/), some of the disabled buttons are still focusable for screenreaders

Reply:
and specifically it says this a little further down the page

> While they are HTML button elements, as described in the accessibility features section above, in order to remain focusable when disabled, they have aria-disabled instead of the HTML disabled attribute.

Reply:
hmm that's very subjective. I would expect to skip over them by default, otherwise that is inconsistent with everything else we currently implement.